src/fsharp/Interactive.DependencyManager/Interactive.DependencyManager.fsproj

```diff
@@ -49,6 +55,7 @@
     <PackageReference Include="System.Reflection.TypeExtensions" Version="$(SystemReflectionTypeExtensionsVersion)" />
     <PackageReference Include="System.Runtime" Version="$(SystemRuntimeVersion)" />
     <PackageReference Include="System.Runtime.InteropServices" Version="$(SystemRuntimeInteropServicesVersion)" />
+    <PackageReference Include="System.Runtime.Loader" Version="$(SystemRuntimeLoaderVersion)" />
```

Comment:
This breaks when using `netstandard2.0` FCS build in .NET Framework process.

Reply:
cc @KevinRansom - we'll need to check if this is inherent to targeting net461 or a separate issue